gk/main.c

```diff
@@ -1408,6 +1413,20 @@ update_ip_hop_count(struct gatekeeper_if *iface, struct ipacket *packet,
 	return 0;
 }
 
+static void
+send_request_to_grantor(struct ipacket *packet, struct gk_fib *fib,
```

Comment:
Add the following comment to this function:
`/* This function is only to be called on flows that are not backed by a flow entry. */`